setup.py

```diff
@@ -545,6 +549,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'singularity': singularity,
     'slack': slack,
     'snowflake': snowflake,
+    'spark': spark,
```

Comment:
Can you also update installation.rsst?
https://airflow.readthedocs.io/en/latest/installation.html

Reply:
Sure. We should automate it in the future though.

Reply:
https://github.com/apache/airflow/issues/8088